keras/backend/__init__.py

```diff
@@ -1,124 +1,156 @@
+from .load_backend import sin
+from .load_backend import cos
+from .load_backend import normalize_batch_in_training
+from .load_backend import batch_normalization
+from .load_backend import concatenate
+from .load_backend import reshape
+from .load_backend import permute_dimensions
+from .load_backend import resize_images
+from .load_backend import resize_volumes
+from .load_backend import repeat_elements
+from .load_backend import repeat
+from .load_backend import arange
+from .load_backend import tile
+from .load_backend import flatten
+from .load_backend import batch_flatten
+from .load_backend import expand_dims
+from .load_backend import squeeze
+from .load_backend import temporal_padding
+from .load_backend import spatial_2d_padding
+from .load_backend import spatial_3d_padding
+from .load_backend import stack
+from .load_backend import one_hot
+from .load_backend import reverse
+from .load_backend import slice
+from .load_backend import get_value
+from .load_backend import batch_get_value
+from .load_backend import set_value
+from .load_backend import batch_set_value
+from .load_backend import print_tensor
+from .load_backend import function
+from .load_backend import gradients
+from .load_backend import stop_gradient
+from .load_backend import rnn
+from .load_backend import switch
+from .load_backend import in_train_phase
+from .load_backend import in_test_phase
+from .load_backend import relu
+from .load_backend import elu
+from .load_backend import softmax
+from .load_backend import softplus
+from .load_backend import softsign
+from .load_backend import categorical_crossentropy
+from .load_backend import sparse_categorical_crossentropy
+from .load_backend import binary_crossentropy
+from .load_backend import sigmoid
+from .load_backend import hard_sigmoid
+from .load_backend import tanh
+from .load_backend import dropout
+from .load_backend import l2_normalize
+from .load_backend import in_top_k
+from .load_backend import conv1d
+from .load_backend import separable_conv1d
+from .load_backend import conv2d
+from .load_backend import separable_conv2d
+from .load_backend import conv2d_transpose
+from .load_backend import depthwise_conv2d
+from .load_backend import conv3d
+from .load_backend import conv3d_transpose
+from .load_backend import pool2d
+from .load_backend import pool3d
+from .load_backend import bias_add
+from .load_backend import random_normal
+from .load_backend import random_uniform
+from .load_backend import random_binomial
+from .load_backend import truncated_normal
+from .load_backend import ctc_label_dense_to_sparse
+from .load_backend import ctc_batch_cost
+from .load_backend import ctc_decode
+from .load_backend import map_fn
+from .load_backend import foldl
+from .load_backend import foldr
+from .load_backend import local_conv1d
+from .load_backend import local_conv2d
+from .load_backend import backend
+from .load_backend import pattern_broadcast
+
+# undocumented
+from .load_backend import name_scope

```

Comment:
This method can be normally replicated in all backends.

Reply:
This method is already implemented in all backends.